### PR TITLE
Issue144 data gen args

### PR DESCRIPTION
**Pull request by dhblum**
Addresses https://github.com/ibpsa/project1-boptest/issues/144.

### CODE REVIEW

**Comment by dhblum** [comment]
@JavierArroyoBastida Can you review these changes and approve before merging?

File: data/data_generator.py [hunk 1]
```diff
@@ -211,33 +211,47 @@ def generate_weather(self,
...
+                      price_solar_thermal_power = 0.0):
+                      
         '''Append the prices for different energy sources.
         There are three different scenarios considered for electricity:
             1. PriceElectricPowerConstant: completely constant price
             2. PriceElectricPowerDynamic: day/night tariff
             3. PriceElectricPowerHighlyDynamic: spot price 
             
-        All prices are expressed in ($/euros)/Kw*h.
+        All prices are expressed in ($/euros)/KW*h.
```
Comment (JavierArroyoBastida) [inline data/data_generator.py]: Tiny detail: maybe kW*h more correct? sometimes I mistake a capital K for the units.

**Comment by JavierArroyoBastida** [comment]
@dhblum I think the wording makes more sense now. Just added a small remark above. Please go ahedad with the merge!

File: data/data_generator.py (hunk 1 above)
Comment (dhblum) [reply data/data_generator.py]: I edited so that everything reads kWh.

**Comment by dhblum** [comment]
Thanks! Will merge after checks.

**Pull request merged**